extensions/security/runtime/src/main/java/io/quarkus/security/runtime/QuarkusSecurityIdentity.java

```diff
@@ -258,9 +303,32 @@ public QuarkusSecurityIdentity build() {
             if (principal == null && !anonymous) {
                 throw new IllegalStateException("Principal is null but anonymous status is false");
             }
+            addPossesedPermissionsChecker();
 
             built = true;
             return new QuarkusSecurityIdentity(this);
         }
+
+        public void addPossesedPermissionsChecker() {
```

Comment:
this should be probably private

Reply:
Yeah, IDE made it public and I did not notice